ipalib/cli.py

```diff
@@ -200,7 +200,7 @@ def print_plain(self, string):
         """
         Print exactly like ``print`` statement would.
         """
-        print(unicode(string))
+        logger.info_cmd(unicode(string))
```

Comment:
While you are on it, kill ``unicode()`` in ``logger.info_cmd()`` commands

Reply:
cli.py is not patched anymore